bedrock/newsletter/views.py

```diff
@@ -362,11 +337,11 @@ def existing(request, token=None):
                 else:
                     title = nstrings['title']
                 description = nstrings.get('description', u'')
-            else:
+            # else:
                 # Firefox Marketplace for Desktop/Android/Firefox OS should be
                 # shorten in the titles
-                title = _(data['title'].replace('Firefox Marketplace for ', ''))
-                description = _(data['description'])
+                # title = _(data['title'].replace('Firefox Marketplace for ', ''))
```

Comment:
I'm not sure how to handle this with Fluent - but I also thought it probably isn't needed today anyway? Thoughts?

Reply:
I think you're right. We should be fine to remove this.